### PR TITLE
Add documentation for approving self-sign-ups

### PR DESCRIPTION
**Pull request by RukshiW**
Resolves https://github.com/wso2-enterprise/choreo/issues/16626

### CODE REVIEW

**Review by RolandHewage — DISMISSED** [review]
Let's update the image with the renamed `Auto-approval` switch. Rest of the content seems fine.

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 1]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+## Approve user accounts
+
+To check the users who try to access your Developer Portal via self-sign-up, you can follow either of the following approaches:
+
+- Enable auto-approval so that every user who signs up is allowed to access your Developer Portal by default.
+- Manually check each sign-up and approval or reject it as required.
+
+### Enable auto-approval
+
+To enable auto-approval, follow the steps given below:
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:118]: Shall we add a statement as follows: " You can use the auto-approval function in Choreo to automatically approve new user registrations on the Developer portal."

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 2]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+1. Click **Settings** in the left navigation menu of the Choreo Console.
+
+2. Click **Self Signups**. 
+
+3. Toggle the **Auto-approval** switch.  This enables your account to automatically approve user accounts that create accounts to access your Developer Portal.
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:124]: If we have this info in the introductory sentence, lets remove "This enables your account to automatically approve user accounts that create accounts to access your Developer Portal."

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 3]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+    ![Auto-approval](../assets/img/administration/self-sign-up/auto-approval.png){.cInlineImage-threeQuarters}
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:126]: Do we need this image?

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 4]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+Once you enable auto-approval, the users can sign in to your Developer Portal and view your APIs and applications soon after they create an account.
+
+### Manually approve/reject user accounts
+
+If you have not enabled auto-approval, you need to follow the procedure given below to manually approve/reject the sign-up:
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:132]: Shall we rephrase it as follows "You can streamline the user registrations by manually approving/rejecting new user registrations. Follow the steps below to manually approve/reject the self-sign-up requests"

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 5]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+1. Once the user creates an account, Choreo sends an email to ask the user to confirm the account. This email is similar to the one in the image below.
+
+    ![Confirm the account](../assets/img/administration/self-sign-up/confirm-account.png){.cInlineImage-half}
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:136]: Do we need this image?

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 6]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+2. The user needs to click **Confirm Account**.
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:138]: Do we need the first two steps at all? If so, we should add them at the bottom, as what a user can expect, or in the first paragraph of this section. I don't think these are part of the flow

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 7]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+3. Click **Settings** in the left navigation menu of the Choreo Console, and then click **Self Signups**. You will see the user account listed for approval as shown below:
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:140]: Let's remove the sentence "You will see the user account listed for approval as shown below:"

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 8]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+    ![Pending approval](../assets/img/administration/self-sign-up/approval-pending.png){.cInlineImage-threeQuarters}
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:142]: Let's not use this image as well.

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 9]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+4. If you want to approve the user account, click **Approve**. If not, click **Reject**.
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:144]: From the account list, to approve a specific user account, click **Approve** for that user account. If you wish to reject, click **Reject**.

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 10]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+    If you approve, the user will receive an email confirming that the user account is approved. This email is similar to the one in the image below.
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:146]: We don't need to include the image of an email. Let's remove that, and the sentence that says the email is similar..

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 11]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+
+    ![User account approved](../assets/img/administration/self-sign-up/user-account-approved.png){.cInlineImage-half}
+
+    If you reject the user account, Choreo sends an email to inform the user that the user account is rejected. Once you reject a user account, that user cannot sign up to your Developer Portal via that account again.
```
Comment (shaniR) [inline en/docs/administer/configure-developer-portal-self-sign-up.md:150]: Can they create another account with the same info again? Does it give info as to why it got rejected?

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 9 above)
Comment (shaniR) [reply en/docs/administer/configure-developer-portal-self-sign-up.md:144]: Can you add a reason as to why it was rejected?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 3 above)
Comment (RukshiW) [reply en/docs/administer/configure-developer-portal-self-sign-up.md]: Removed it

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 6 above)
Comment (RukshiW) [reply en/docs/administer/configure-developer-portal-self-sign-up.md]: Moved them to the first paragraph and rewrote this section. Please check.

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 10 above)
Comment (RukshiW) [reply en/docs/administer/configure-developer-portal-self-sign-up.md]: Removed it.

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 7 above)
Comment (RukshiW) [reply en/docs/administer/configure-developer-portal-self-sign-up.md]: Why remove it. This informs the users where they can find the user accounts to manually approve. Or do you mean remove the image?

Made some changes there.

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 8 above)
Comment (RukshiW) [reply en/docs/administer/configure-developer-portal-self-sign-up.md]: Removed it.

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 12]
```diff
@@ -60,12 +60,15 @@ This page walks you through the steps to configure self-sign-up to the Developer
 9. Click **Update**.
 10. To add user attributes, follow these steps:
 
-    1. Click the **User Attributes** tab and click **+ Add User Attribute**.
-    2. Select **Email**, **First Name**, and **Last Name** as the attributes to share with the application.
-    3. Click **Save**.
-    4. Select **Email** as a mandatory attribute and click **Update**.
+    1. Click the **User Attributes** tab.
+    2. Select user attributes as follows:
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/administer/configure-developer-portal-self-sign-up.md:64]: In that case shall we move the image near the relevant text?

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 11 above)
Comment (shaniR) [reply en/docs/administer/configure-developer-portal-self-sign-up.md:140]: Ack.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/administer/configure-developer-portal-self-sign-up.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/administer/configure-developer-portal-self-sign-up.md:64]: The text with the steps is right below the image.

File: en/docs/administer/configure-developer-portal-self-sign-up.md [hunk 13]
```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
...
+![Create an account](../assets/img/administration/self-sign-up/create-an-account.png){.cInlineImage-half} 
+
+## Approve user accounts
+
+To check the users who try to access your Developer Portal via self-sign-up, you can follow either of the following approaches:
+
+- Enable auto-approval so that every user who signs up is allowed to access your Developer Portal by default.
+- Manually check each sign-up and approval or reject it as required.
+
+### Enable auto-approval
```
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/administer/configure-developer-portal-self-sign-up.md]: Done

**Review by nilminiwso2 — APPROVED** [review]

**Review by shaniR — APPROVED** [review]

**Pull request merged**